nix-repl.el

```diff
@@ -149,6 +149,7 @@ guarantees they will be grabbed in a single call."
 See `completion-at-point-functions'."
   (save-excursion
     (let* ((proc (get-buffer-process (current-buffer)))
+           (process-live-p proc)
```

Comment:
I don't think this does what you think... :D

Reply:
Egads, that was a silly mistake. Thank you.